flink-connector-aws/flink-connector-aws-kinesis-streams/pom.xml

```diff
@@ -122,6 +122,12 @@ under the License.
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>s3</artifactId>
+            <scope>test</scope>
+        </dependency>
+
```

Comment:
Why do we test using s3? Can we use kinesis instead?

Reply:
We are actually testing Kinesis only. The S3 dependency is required because the LocalStack container in the connector base arbitrarily creates an S3 bucket and lists its contents to test readiness. [LocalstackContainer.java#L61-L83](https://github.com/apache/flink-connector-aws/blob/7fbca5566c02e3b2c581c9fcdb6c3cc2b633502a/flink-connector-aws-base/src/test/java/org/apache/flink/connector/aws/testutils/LocalstackContainer.java#L61-L83)

Reply:
Right - thanks!